pkgs/applications/misc/rpicam-apps/default.nix

```
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, meson
+, ninja
+, pkg-config
+, boost
+, ffmpeg
+, libcamera
+, libdrm
+, libexif
+, libjpeg
+, libpng
+, libtiff
+}:
+
+stdenv.mkDerivation rec {
```

You should use the `finalAttrs` pattern here so that overriding is more effective.

https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
```suggestion
stdenv.mkDerivation (finalAttrs: {
```